definitions/variable/macro-economy/trade.yaml

```diff
@@ -8,3 +8,54 @@
     unit: billion USD_2010/yr
     tier: 1
     min: 0
+- Trade|Primary Energy|Biomass [Value]:
```

Comment:
Are we sure we want to treat "trade losses" like this? That means the summation checks `World = Sum(Regions)` should be disabled here?